monkestation/code/modules/antagonists/borers/code/evolution/evolution_datum.dm

```diff
@@ -0,0 +1,39 @@
+/datum/borer_evolution
```

Comment:
ngl it'd prolly be a good idea to have like a `var/list/action_types` and just automatically add any actions from that list on `on_evolve`

Reply:
yeah, probably gonna add somethin for action types. And added chemicals for simplicity